src/messages.mts

```diff
@@ -1,48 +1,54 @@
-// @ts-nocheck
-
-import { surroundingAgent, inspect, Value } from './index.mts';
+import type { ArrayBufferObject, FunctionObject } from './abstract-ops/all.mts';
+import type { AbstractModuleRecord } from './modules.mts';
+import type {
+  BooleanValue, JSStringValue, NumberValue, ObjectValue, PropertyKeyValue,
+} from './value.mts';
+import { inspect, PrivateName, Value } from './index.mts';
 
 function i(V: unknown) {
```

Comment:
I'm planning to get rid of `inspect` in error throwing, or improving it with a side-effect-free way of this. It used to cause test262 failure in https://github.com/engine262/engine262/pull/277/files#diff-e02b9e5bf3aa21bec763c8430087794b58a0f38acab9c15a3882825bf115ce2aL59

Reply:
Yeah, I plan to fix `inspect` to be side‑effect free (similar to <https://github.com/engine262/engine262/pull/117>, but better)